internal/schemautil/service.go

```diff
@@ -572,6 +594,20 @@ func getDefaultDiskSpaceIfNotSet(ctx context.Context, d *schema.ResourceData, cl
 	return diskSpace, nil
 }
 
+func getTechnicalEmailsForTerraform(d *schema.ResourceData, field string, s *aiven.Service) *schema.Set {
+	_, existsBefore := d.GetOk(field)
+	if !existsBefore && len(s.TechnicalEmails) == 0 {
```

Comment:
nit: technically "before" is `s.TechnicalEmails`. It is "now" or more conventional is "ok"